packages/kit/src/core/sync/create_manifest_data/index.js

```diff
@@ -444,7 +444,7 @@ function list_files(dir, path = '', files = []) {
 		.forEach((file) => {
 			const joined = path ? `${path}/${file.name}` : file.name;
 
-			if (file.isDirectory()) {
+			if (file.isDirectory() || file.isSymbolicLink()) {
```

Comment:
This might be a symlink to a file and not a directory, right? Don't we need to check what it is that this is a symlink to?

Reply:
Nice catch, yes that would have failed on a file symlink.